UABEAvalonia/Forms/MainWindow.axaml

```diff
@@ -4,10 +4,13 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         mc:Ignorable="d" d:DesignWidth="400" d:DesignHeight="150"
         Width="400" Height="150"
+        MinWidth="400" MinHeight="150"
```

Comment:
Sometimes there can be really long names in bundles, so I'd rather not have this set at a static width.

Reply:
it makes sense, I removed the 2 lines